Automattic/wp-calypso
# AI Blogging prompts: more resiliency

This PR introduces some resiliency in ai blogging prompts.

More data:
p1700421670206969-slack-C03N25JPCE4


## Testing Instructions

* Introduce a failure in the endpoint or return false
```
diff --git a/wp-content/rest-api-plugins/endpoints/ai-blogging.php b/wp-content/rest-api-plugins/endpoints/ai-blogging.php
index 8f9a963b0c..a38d39bc50 100644
--- a/wp-content/rest-api-plugins/endpoints/ai-blogging.php
+++ b/wp-content/rest-api-plugins/endpoints/ai-blogging.php
@@ -26,6 +26,7 @@ class WPCOM_REST_API_V2_Endpoint_AI_Blogging extends WPCOM_REST_API_V2_Endpoint_
        }
 
        public function permission_callback() {
+return false;
                return (
                        is_automattician() && // Only automatticians for now.
                        current_user_can( 'publish_posts' )
```
* Clear cached data (inspector -> application -> indexeddb -> calypso_store -> clear)
* Load/reload home
* See no console errors
* See it showing only the regular prompts

## Pre-merge Checklist
